index.css

```diff
@@ -462,29 +535,392 @@ label {
+  }
+
+  .about_section_align1 img {
+    padding: 10px 0 0 0;
+    background-color: #f7f7f9;
+    border: 1px solid none;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .about_self_section p {
+    color: #344563;
+    font-size: 16px;
+    font-size: 1.44vw;
+    line-height: 24px;
+    font-weight: 400;
+    text-align: justify;
+    justify-content: center;
+  }
+
+  .show1 {
+    display: none;
+  }
+
+  .ul1 {
+    margin: 15px 0 10px 0;
+  }
+
+  .article1div p,
+  .article2div p,
+  .article3div p,
+  .article4div p {
+    width: 100%;
+    height: 50px;
+    text-align: left;
+    color: #344563;
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+    justify-content: center;
+    gap: 25px;
+    margin-top: 10px;
+    padding-bottom: 120px;
+    padding-top: 40px;
+  }
+
+  .font2 {
+    width: 75px;
+    height: 24px;
+    color: #344563;
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+    padding: 1px 0;
+  }
+
+  .font3 {
+    width: 120px;
+    color: #6b778c;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    padding-left: 15px;
+  }
+
+  .font4 {
+    width: 295px;
+    color: #6070ff;
+    font-size: 12px;
+    align-items: left;
+    font-weight: 500;
+    row-gap: 25px;
+    padding: 16px 2px 2px 5px;
+  }
+
+  .about_self_section {
+    display: flex;
+    justify-content: center;
+    gap: 15px;
+    margin: auto;
+  }
+
+  .about_width1 {
+    width: 40%;
+  }
+
+  .font5 {
+    color: #253858;
+    line-height: 20px;
+    font-size: 15px;
+    align-items: left;
+    font-weight: 500;
+    gap: 25px;
+    padding: 16px 2px 2px 5px;
+  }
+
+  .about_self_section h3 {
+    padding: 0 0 20px 0;
+    width: 170px;
+    line-height: 40px;
+  }
+
+  .about_section_flex2 {
+    padding: 0;
+  }
+
+  .about_section_flex1 {
+    margin-top: -30px;
+  }
+
+  .about_self_section .social2 {
+    padding: 20px 0;
+  }
+
+  .about_section_flex3 {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    gap: 5px;
+    justify-content: flex-start;
+    text-align: left;
+  }
+
+  .about_section_align1 {
+    padding: 10px 2px 10px 2px;
+    background-color: #f7f7f9;
+    border: 1px solid none;
+    border-radius: 10px;
     display: flex;
     flex-direction: column;
-    align-items: center;
+    justify-content: flex-start;
+  }
+
+  .font9 {
+    width: 295px;
+    color: #344563;
+    font-size: 20px;
+    align-items: left;
+    font-weight: 500;
+    line-height: 24px;
+  }
+
+  .contact_section {
+    padding: 150px 0 20px 0;
+    border-top-left-radius: 100px;
+    display: flex;
+    justify-content: center;
+    background-color: #6070ff;
+    background-image: url("images/Header bg.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: right 5px bottom 10%;
+  }
+
+  .contact_form {
+    width: 50%;
+    margin: auto;
+  }
+
```

Comment:
Excelent work making the changes requested by the previous reviewer 🥇 

But there's some changes in with you need to still working 

- [ ] Making responsive your portfolio, means that no matter in which size you put it, elements and images will be displayed correctly. In this case, kindly, fix your background image to fit the max-content of your screen as it is on Figma's design

![image](https://user-images.githubusercontent.com/93630700/213876333-4c17632f-4eda-4a30-b57b-b63f6998a43e.png)

Reply:
Fixed the contact form according to the review.